Doppler.Sap/Factory/BillingRequestHandler.cs

```diff
@@ -98,7 +98,7 @@ private async Task<SapTaskResult> SendIncomingPaymentToSap(SapServiceConfig serv
 
             if (!sapResponse.IsSuccessStatusCode)
             {
-                _logger.LogError($"Incoming Payment could'n create to SAP because exists an error: '{sapResponse.Content.ReadAsStringAsync()}'.");
+                _logger.LogError($"Incoming Payment could'n create to SAP because exists an error: '{await sapResponse.Content.ReadAsStringAsync()}'.");
```

Comment:
Mmm, it is weird, because an async operation could fail.

Reply:
It's better if we use sapResponse.Content.ReadAsStringAsync().Result? We have a doubt with that

Reply:
We have talked about it.

Using `.Result` is not a good idea inside an `async` method ([and you have to think twice in a not async method](https://medium.com/rubrikkgroup/understanding-async-avoiding-deadlocks-e41f8f2c6f5d)).